x/oracle/types/params.go

```diff
@@ -44,6 +44,7 @@ var (
 		asset.Registry.Pair(denoms.ATOM, denoms.USD),
 		asset.Registry.Pair(denoms.USDC, denoms.USD),
 		asset.Registry.Pair(denoms.USDT, denoms.USD),
+		asset.Registry.Pair(denoms.USDA, denoms.USD),
```

Comment:
usda:uusd is a pair with different orders of magnitude. I would recommend doing uusda:uusd to keep it in micro-units.